src/container/container_iv.c

```diff
@@ -975,7 +975,7 @@ cont_iv_prop_g2l(struct cont_iv_prop *iv_prop, daos_prop_t *prop)
 }
 
 int
-cont_iv_prop_update(void *ns, uuid_t cont_hdl_uuid, uuid_t cont_uuid,
+cont_iv_prop_update(void *ns, uuid_t iv_key_uuid, uuid_t cont_uuid,
```

Comment:
Is this just a name change for clarity or do the semantics change in the code further down the call stack?

Reply:
Just a name change. Just changed it because it's not always the container handle. There are some comments elsewhere in the code where they say it should be the container UUID for props (and that did seem to work in this case), although it used the handle UUID in cont_open. I experimented with that, and for cont_open it did need the handle UUID. I'm not sure of the reasoning behind that - was hoping to get some input from @wangdi1 if he gets a chance.